src/testHelper.js

```diff
@@ -0,0 +1,70 @@
+import dotenv from 'dotenv';
+import {createTestUser, removeTestUser} from '@webex/test-users';
+import Webex from 'webex';
+
+// Used to get WEBEX_CLIENT_ID & WEBEX_CLIENT_SECRET for test user creation access
+dotenv.config();
+
+const URL_CONVO_SERVICE_INTEGRATION = 'https://conversation-intb.ciscospark.com/conversation/api/v1';
+const URL_ID_BROKER_INTEGRATION = 'https://idbrokerbts.webex.com';
+const URL_U2C_INTEGRATION = 'https://u2c-intb.ciscospark.com/u2c/api/v1';
+const SCOPE =
+  'Identity:OAuthClient webexsquare:get_conversation webexsquare:admin Identity:SCIM spark:people_read spark:rooms_read spark:rooms_write spark:memberships_read spark:memberships_write spark:messages_read spark:messages_write spark:applications_read spark:applications_write spark:teams_read spark:teams_write spark:team_memberships_read spark:team_memberships_write spark:bots_read spark:bots_write spark:kms';
+
+/**
+ * Creates a test user in the integration webex environment.
+ *
+ * @export
```

Comment:
Is `@export` a JS Doc tag? I couldn't find it.
There is _@exports_, but JS Docs's [example section](https://jsdoc.app/tags-exports.html) says it's totally unnecessary

Reply:
My jsdoc generator plugin adds them by default. I will remove if they bother you.
https://marketplace.visualstudio.com/items?itemName=joelday.docthis

Reply:
I see, weird that they add a tag that doesn't exist.
Let's keep our doc strings minimal, so i'd appreciate if you can remove them 👍

Reply:
They have already been removed 😄